scripts/qa_crowbarsetup.sh

```diff
@@ -2517,7 +2517,13 @@ function onadmin_addupdaterepo()
     if [[ -n "$UPDATEREPOS" ]]; then
         local repo
         for repo in ${UPDATEREPOS//+/ } ; do
-            wget --progress=dot:mega -r --directory-prefix $UPR --no-parent --no-clobber --accept x86_64.rpm,noarch.rpm $repo || exit 8
+            wget --progress=dot:mega \
+                -r --directory-prefix $UPR \
+                --no-parent \
+                --no-clobber \
+                --accept x86_64.rpm,noarch.rpm \
+                ${repo%/}/ \
```

Comment:
what's the advantage? the other one works, and this one is a bit harder to decipher :-)